test/unit/channel_factory_test.js

```diff
@@ -90,16 +90,6 @@ for (const grpcLibName of ['grpc', '@grpc/grpc-js']) {
           assert.doesNotThrow(() => {
             new grpcGcp.GcpChannelFactory('hostname', insecureCreds, {key: 5});
           });
-          assert.throws(() => {
-            new grpcGcp.GcpChannelFactory('hostname', insecureCreds, {
-              key: null,
-            });
-          });
-          assert.throws(() => {
-            new grpcGcp.GcpChannelFactory('hostname', insecureCreds, {
-              key: new Date(),
-            });
-          });
```

Comment:
Why are we removing these?

Reply:
They stopped being valid when this project moved to grpc-js. I'm guessing the previously the grpc-core based impl would throw errors on non-string values, but grpc-js does not